### PR TITLE
Adding github workflow for contributing to helm

### PR DESCRIPTION
**Pull request by truongnh1992**
Describe detail step-by-step of using github to contribute to
helm project.

Co-Authored-By: Dao Cong Tien tiendc@vn.fujitsu.com
Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

### CODE REVIEW

**Review by mattfarina — COMMENTED** [review]
Thanks for your interest in our contributions process. This type of content should be in the `CONTRIBUTING.md` file. That is the standard file name GitHub expects. Some of this is already in that file. Would you mind trying to make the improvements there.

**Comment by truongnh1992** [comment]
> Thanks for your interest in our contributions process. This type of content should be in the `CONTRIBUTING.md` file. That is the standard file name GitHub expects. Some of this is already in that file. Would you mind trying to make the improvements there.

I've just resolved your comment, the content of `CONTRIBUTING.md` has been updated.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -172,10 +172,49 @@ contributing to Helm. All issue types follow the same general lifecycle. Differe
...
+6. Make any change on the branch `new-feature`  then build and test your codes.
+7. Include in what will be committed.
+```sh
+$ git add <file>
+```
+8. Use sign-off when making each of your commits (see [above](#sign-your-work)).
    If you forgot to sign some commits that are part of the contribution, you can ask [git to rewrite your commit history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
-1. Submit a pull request.
+```sh
+$ git commit -s
```
Comment (bacongobbler) [inline CONTRIBUTING.md]: Woudl you mind using the long form here? `git commit --signoff` is more memorable for first-timers in general.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by bacongobbler** [comment]
Why were the file permissions on CONTRIBUTING.md changed from 644 to 755?

**Comment by truongnh1992** [comment]
> Why were the file permissions on CONTRIBUTING.md changed from 100644 to 100755?

Thank you for your review. I've already resolved all of your changes request.

**Comment by truongnh1992** [comment]
@mattfarina could you help me to review this change, please.

**Review by michelleN — APPROVED** [review]

**Review by mattfarina — APPROVED** [review]

**Comment by mattfarina** [comment]
@truongnh1992 Thanks for the PR. We discussed it in the Helm Dev Call. It was a useful conversation.

**Comment by truongnh1992** [comment]
> 
> 
> @truongnh1992 Thanks for the PR. We discussed it in the Helm Dev Call. It was a useful conversation.

You're welcome. I'm willing to contribute to opensource project. By the way, how can I become a member of helm org on Github?

**Comment by mattfarina** [comment]
@truongnh1992 You can contribute without being a member of the Helm org on GitHub. The members of the org are those who have been or currently are maintaining a project that is part of Helm.

**Pull request merged**